lib/routes/airlevel/namespace.ts

```diff
@@ -0,0 +1,14 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'AirLevel',
+    url: 'air-level.com',
+    description: `
+    * 可以订阅每个城市的空气质量，按照拼音订阅
+    * 支持订阅每天的实时排名
```

Comment:
Indent with either 0 or 2 spaces or the markdown content won't be rendered properly.
Use `-` for a list.